src/web/components/Cards/Card/Card.tsx

```diff
@@ -129,7 +130,14 @@ export const Card = ({
                                             />
                                         </StandfirstWrapper>
                                     )}
-                                </>
+                                    {webPublicationDate && (
```

Comment:
It's possibly because it's late in the day but I'm not sure I follow. `webPublicationDate` is already optional on `CardType` so it can it can not be passed in. If it is, it causes the age tag to show.

...oh, do you mean pass it into `CardAge`? Hmm, so my preference is for presentation components to be fairly stupid and have a layout/decision making wrapper. That said, this is one of those nice sounding ideas that often dies when thrown at the brick wall of reality though!

Reply:
Ah how I miss Container Components sometimes. That sounds reasonable though, happy to concede.